apps/app/src/updater_impl.rs

```diff
@@ -0,0 +1,122 @@
+use crate::api::Result;
+use std::sync::{Arc, Mutex};
+use tauri::http::HeaderValue;
+use tauri::http::header::ACCEPT;
+use tauri::{Manager, ResourceId, Runtime, Webview};
+use tauri_plugin_http::reqwest;
+use tauri_plugin_http::reqwest::ClientBuilder;
+use tauri_plugin_updater::Error;
+use tauri_plugin_updater::Update;
+use theseus::{LoadingBarType, emit_loading, init_loading};
+use tokio::time::Instant;
+
+const UPDATER_USER_AGENT: &str =
+    concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_PKG_VERSION"));
```

Comment:
I think this user agent string should match the format used by the app elsewhere, namely in the `app-lib` package:

https://github.com/modrinth/code/blob/e15fa35bad471836377b7ab34c0c83830dec9cbc/packages/app-lib/src/util/fetch.rs#L23-L26

Reply:
I used this User Agent, as this is the agent used to actually download the update. However, the downloading User Agent can be overridden, which I can do.

Reply:
I see, fair enough. Feel free to go with the approach you think would work best for filtering/tagging update-related traffic then :+1: